context/.prettierrc.yml

```diff
@@ -0,0 +1,5 @@
+semi: true
+trailingComma: "all"
+singleQuote: true
+printWidth": 70
```

Comment:
```suggestion
printWidth: 70
```

Reply:
Also, would you like this to be higher? Perhaps look at the reformatted code and see if it's doing too much wrapping?

Reply:
Yeah, it was much too drastic. I increased it to 120.